src/amcrest/snapshot.py

```diff
@@ -20,10 +20,9 @@
 
 
 class Snapshot(object):
-    def __get_config(self, config_name):
-        ret = self.command(
-            'configManager.cgi?action=getConfig&name={0}'.format(config_name)
-        )
+    def __get_config(self, config_name, channel=1):
+        cmd = 'configManager.cgi?action=getConfig&name={0}&channel={1}'
```

Comment:
Are you sure this works? According to the doc this command does not accept a channel option, but rather returns an array containing data for each channel.

Reply:
indeed, not relevant here.